spec/support/factory_girl.rb

```diff
@@ -0,0 +1,23 @@
+RSpec.configure do |config|
+  config.include FactoryGirl::Syntax::Methods
+  config.before :suite do
+    DatabaseCleaner.clean_with :truncation
+  end
+
+  config.before :each do
+    DatabaseCleaner.strategy = :transaction
+  end
+
+  config.before(:each, js: true) do
+    DatabaseCleaner.strategy = :truncation
+  end
+
+  config.before :each do
+    DatabaseCleaner.start
+    FactoryGirl.reload
```

Comment:
It seems like the FactoryGirl.reload is an [antipattern](https://robots.thoughtbot.com/getting-sequential-a-look-into-a-factory-girl).  It seems like the goal should be to assign names when we need them and let the sequences do their thing when the specific name isn't crucial.  Thoughts? @Robbie-Smith